### PR TITLE
Feedback 10 07 mdp

### PR DESCRIPTION
**Pull request by d-morrison**

### CODE REVIEW

File: vignettes/articles/subtyping-and-staging.qmd
Comment (d-morrison) [inline vignettes/articles/subtyping-and-staging.qmd:1]: @mponzini are you still using `subtyping-and-staging.qmd`?

Comment (mponzini) [reply vignettes/articles/subtyping-and-staging.qmd:1]: No, but it looks like it's still in my vignettes/articles/ folder?

Comment (d-morrison) [reply vignettes/articles/subtyping-and-staging.qmd:1]: Ok, let's delete it again? It's producing some spell-check errors, and we can always retrieve it from git history if it turns out we need it.

Comment (mponzini) [reply vignettes/articles/subtyping-and-staging.qmd:1]: killed it and pushed

**Pull request merged**